browser/ui/webui/brave_wallet/panel_handler/wallet_panel_handler.cc

```diff
@@ -7,7 +7,25 @@
 
 #include <utility>
 
+#include "base/json/json_reader.h"
+#include "brave/browser/brave_wallet/brave_wallet_tab_helper.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_utils.h"
+#include "brave/components/brave_wallet/browser/eth_response_parser.h"
+#include "brave/components/brave_wallet/browser/pref_names.h"
 #include "brave/components/permissions/contexts/brave_ethereum_permission_context.h"
+#include "components/prefs/scoped_user_pref_update.h"
+#include "components/user_prefs/user_prefs.h"
+#include "content/public/browser/browser_context.h"
+#include "content/public/browser/web_contents.h"
+
+namespace {
+// From https://eips.ethereum.org/EIPS/eip-3085
+const char kRejectedResponse[] = R"(
+    "error": {
+    "code": 4001,
```

Comment:
better to move that into `brave/components/brave_wallet/common/web3_provider_constants.h`

Reply:
done